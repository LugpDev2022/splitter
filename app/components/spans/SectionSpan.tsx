interface sectionSpanProps {
  children: React.ReactNode;
}

const SectionSpan: React.FC<sectionSpanProps> = ({ children }) => (
  <span
    className='font-bold text-lg w-full block'
    style={{ color: 'hsl(186, 14%, 43%)' }}
  >
    {children}
  </span>
);

export default SectionSpan;
