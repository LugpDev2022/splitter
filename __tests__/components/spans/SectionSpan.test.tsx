import SectionSpan from '@/app/components/spans/SectionSpan';
import { render, screen } from '@testing-library/react';

describe('Tests on <SectionSpan />', () => {
  test('should match snapshot', () => {
    const { container } = render(<SectionSpan>Test</SectionSpan>);
    expect(container).toMatchSnapshot();
  });

  test('should show the given quantity correctly', () => {
    render(<SectionSpan>Test</SectionSpan>);
    const span = screen.getByText('Test');
    expect(span).toBeTruthy();
  });
});
