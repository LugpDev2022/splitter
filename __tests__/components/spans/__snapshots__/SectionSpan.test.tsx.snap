// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on <SectionSpan /> should match snapshot 1`] = `
<div>
  <span
    class="font-bold text-lg text-[#5e7a7d]"
  >
    Test
  </span>
</div>
`;
